newsfragments/472.changed.md

```diff
@@ -0,0 +1 @@
+Synapse: Upgrade from v1.129.0 to v1.130.0.
```

Comment:
We've now said that component version upgrade release notes should a) include highlights where appropriate and b) include links to the upstream project's complete release notes